tests/core/pyspec/eth2spec/test/electra/unittests/test_config_invariants.py

```diff
@@ -8,5 +8,15 @@
 @with_electra_and_later
 @spec_test
 @single_phase
-def test_withdrawals(spec):
+def test_withdrawal_requests(spec):
     assert spec.MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD < spec.MAX_WITHDRAWALS_PER_PAYLOAD
```

Comment:
I suppose this should be updated to `spec.MAX_PENDING_PARTIALS_PER_WITHDRAWALS_SWEEP < spec.MAX_WITHDRAWALS_PER_PAYLOAD` — to ensure there is always a room for the sweep

Reply:
yeah, this is where it gets confusing 😅 

I interpreted this test as just ensuring that the number of EL requests would be less than the number of withdrawals, as a bit of a sanity check

but if this existing test meant to cover this use case then we can consolidate into 1

the confusing bit for me is using this constant to talk about "per payload" when it really has nothing to do with the payload, it is more about how we handle processing of withdrawals (regardless of source)